MKOB/kobwindow.py

```diff
@@ -182,6 +175,22 @@ def __init__(self, root, VERSION):
         self.varCodeSenderLoop.set(kc.CodeSenderLoop)
         self.varWireNo.set(kc.WireNo)
 
+##        print("\n\n----------\n\nget_widget_attributes")
+##        get_widget_attributes(pwd2)
+
+def get_widget_attributes(widget):
+    """display widget attributes for debugging GUI"""
+    all_widgets = widget.winfo_children()
+    print("\n", all_widgets)
+    for widg in all_widgets:
+        print("\nWidget Name: {}".format(widg.winfo_class()))
+        keys = widg.keys()
+        for key in keys:
+            print("Attribute: {:<20}".format(key), end=' ')
+            value = widg[key]
+            vtype = type(value)
+            print("Type: {:<30} Value: {}".format(str(vtype), value))
+
 ## TAKE CARE OF THESE
```

Comment:
If this is important - can you use `TODO:` as a lead-in so we can consistently search for it (please many IDE's automatically highlight/index those contents).

Reply:
Thanks for the suggestion! I'll follow that convention in the future.